IDEAS/Buildings/Components/Zone.mo

```diff
@@ -6,46 +6,92 @@ model Zone "thermal building zone"
   extends IDEAS.Fluid.Interfaces.LumpedVolumeDeclarations(
     energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
     redeclare replaceable package Medium = IDEAS.Media.Air,
-    final mSenFac = corrCV);
+    final mSenFac = 1);
```

Comment:
Since this value is no longer used. To set corrCV you should now set mSenFac in the zone air model, since this is in my opinion a property of the zone air model and not a property of the zone itself. Imagine for instance that in the future we may have a CFD model as a zone air model. The use of a corrCV value would then be quite ridiculous.